onnxruntime/core/mlas/inc/mlas.h

```diff
@@ -1825,3 +1825,35 @@ MlasNhwcAvgPool(
     );
 
 #endif
+
+struct MlasFlashAttentionThreadedArgs {
+    int batch_size;
+    int num_heads;
+    int q_sequence_length;
+    int kv_sequence_length;
+    int qk_head_size;
+    int v_head_size;
+    int block_size_q;
```

Comment:
nit: I saw _q and q_ in block_size_q and q_sequence_length. Could we have more consistent naming (like change this to q_block_size)?

Reply:
Done.